### PR TITLE
Hotpath wasm article fixes

### PR DESCRIPTION
**Pull request by surma**
**Target Live Date:** ASAP

- [ ] This has been reviewed and approved by (NAME)
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

### CODE REVIEW

File: src/content/en/updates/2019/02/hotpath-with-wasm.md [hunk 1]
```diff
@@ -71,12 +71,12 @@ WebAssembly can help.
...
-benefit that WebAssembly offers is _predictable_ performance, even across
-browsers. The strict typing and low-level architecture allows for stronger
-assumptions and more optimizations during compilation. The function above is a
-prime candidate for WebAssembly. But how do you turn hot path written in
-JavaScript into WebAssembly?
+However, for JavaScript this performance can only be reached on the "fast path",
+and it's often tricky to stay on that "fast path". One key benefit that
+WebAssembly offers is predictable performance, even across browsers. The strict
+typing and low-level architecture allows the compiler to make stronger
+assumption so that WebAssembly code only has to be optimized once and will
```
Comment (mathiasbynens) [inline src/content/en/updates/2019/02/hotpath-with-wasm.md]: `s/to make strong assumption/to have stronger guarantees/`

There’s no need to assume at all!

File: src/content/en/updates/2019/02/hotpath-with-wasm.md [hunk 2]
```diff
@@ -493,8 +497,21 @@ language. The AssemblyScript team has been very responsive and is actively
 working on improving their toolchain. We will definitely keep an eye on
 AssemblyScript in the future.
 
+## Update: Rust {: #update-rust }
+
+After publishing this article, [Nick Fitzgerald](https://twitter.com/fitzgen)
+from the Rust team pointed us to their excellent Rust WASM book, which contains
```
Comment (mathiasbynens) [inline src/content/en/updates/2019/02/hotpath-with-wasm.md]: nit: `s/WASM/Wasm/`

**Comment by surma** [comment]
Thank you very much, @mathiasbynens :)

**Comment by mathiasbynens** [comment]
cc @gahaas

**Comment by gahaas** [comment]
LGTM

**Pull request merged**